packages/fxa-content-server/app/scripts/views/sign_up_password.js

```diff
@@ -40,6 +40,15 @@ const SignUpPasswordView = FormView.extend({
     'click #use-different': preventDefaultThen('useDifferentAccount'),
   }),
 
+  _isEmailRelayDomain(email) {
+    // Checks to see if the email is a Firefox Relay email mask. Current masks are
+    // @mozmail.com
+    // @relay.firefox.com
+    // @<any sub>.mozmail.com
+    const relayMail = /@([a-zA-Z0-9.-]+\.)?(mozmail|relay\.firefox)\.(com)$/i;
+    return relayMail.test(email);
+  },
```

Comment:
This appears to be identical to the function in index.js above.  Can we have just one of these?